react/ui-components/src/hoc/InboxSearchComposerReducer.js

```
@@ -91,13 +93,23 @@ const reducer = (state, action) => {
         default:
           break;
       }
+      break;
     case 'obj':
-        const {updatedState} = action
-        return updatedState
-
+      const {updatedState} = action
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)